configuration/pih/htmlforms/hiv/section-hiv-history.xml

```diff
@@ -711,6 +721,42 @@
         </section>
     </includeIf>
 
+    <section id="contacts" sectionTag="fieldset" headerTag="legend" headerStyle="title"
+            headerCode="pihcore.contacts">
+        <repeat with="['1'],['2'],['3'],['4'],['5']">
+            <div class="section-container mb-4 flex-questions">
+                <obsgroup groupingConceptId="PIH:Contact construct">
+                    <div>
+                        <obs conceptId="PIH:RELATIONSHIP OF RELATIVE TO PATIENT"
+                         labelCode="pihcore.contactRelationship"
+                         answerConceptIds="PIH:PARTNER OR SPOUSE,PIH:GUARDIAN RELATIONSHIP TO CHILD,PIH:SIBLING,PIH:OTHER RELATIVE"/>
+                    </div>
+                    <div>
+                        <obs conceptId="PIH:FIRST NAME" labelCode="pihcore.firstName"/>
+                    </div>
+                    <div>
+                        <obs conceptId="PIH:LAST NAME" labelCode="pihcore.lastName"/>
+                    </div>
+                    <div>
+                        <obs conceptId="PIH:AGE" labelCode="pihcore.age"/>
+                    </div>
+                    <div>
+                        <obs conceptId="CIEL:163533"
+                             labelCode="pihcore.dead"
+                             answerConceptIds="PIH:ALIVE,PIH:DEATH,PIH:UNKNOWN"
+                             style="checkbox"/>
+                    </div>
+                    <div>
+                        <obs conceptId="PIH:RESULT OF HIV TEST"
+                             labelCode="pihcore.hiv.serologicalStatus"
+                             answerConceptIds="PIH:UNKNOWN,PIH:POSITIVE,PIH:NEGATIVE"
```

Comment:
Not sure of the subtle difference between unknown and Indeterminate, but the paper form and concept have "Indeterminate", positive and negative.  I would modify to use Indeterminate.

Reply:
The paper form has "inc.," which is short for "inconnue," which  which is French for "unknown."

The difference between "unknown" and "indeterminate," as I understand it, is that indeterminate is a kind of reading from a test, whereas unknown could signify an indeterminate test, no test done, or whatever else.

Reply:
^^ @lnball